bin/jade.js

```diff
@@ -144,6 +130,51 @@ if (files.length) {
 }
 
 /**
+ * Watch for changes on path
+ * Renders base if specified, otherwise renders path
+ * If watched path was moved or removed, unwatches the path
+ */
+function watchFile(path, base) {
+  path = normalize(path);
```

Comment:
Personally I don't really care what we log, @TimothyGu any thoughts? I guess normalize should still work.